iana-mediatypes/oci-mediatype.md

```diff
@@ -0,0 +1,66 @@
+**Name:** Steve Lasker
+
+**Email:** tob@opencontainers.org
+
+**Media type name:** application
+
+**Media subtype name:** application/vnd.oci.image.manifest.v1+json
```

Comment:
> IANA: Given that this is a JSON format - this wasn't clear in the original registration - the recommendation is that the media type have +json suffix. Note that this isn't required, but it is strongly recommended.  

Initially submitted as a root mediatype. Changed from `application/vnd.oci.image.manifest` to `application/vnd.oci.image.manifest.v1+json` to reflect no assumptions are made about parent ownership.
This also accounts for the json extension feedback.